pages/linux/qm-guest-passwd.md

```diff
@@ -0,0 +1,12 @@
+# qm guest passwd
+
+> Sets the password for the given user on QEMU/KVM Virtual Machine Manager.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Set the password for the provided user in a vm:
+
+`qm guest passwd {{vm_id}} {{username}}`
+
+- Specify if the password has already been encrypted using crypt():
```

Comment:
Can you explain this command to me?

Reply:
It lets you interactively set the password of a user in a vm

Reply:
This way?
```suggestion
- Set the password for a specific user in a virtual machine interactively:
```

Reply:
ah sorry for the miscommunication, using the `--crypted 1` flag lets you use a password that has already been hashed using crypt()

Reply:
and how you use that hash?

Reply:
Usually, the password is hashed using this https://manned.org/crypt.3, and then the hashed output is passed interactively through the terminal when prompted for a password. It skips using the default encryption provided by `qm guest password` and instead stores the provided password hash directly.
kind of an edge case tbh but nice to have I guess

Reply:
```suggestion
- Set an already hashed password for a specific user in a virtual machine interactively:
```